morpheus/_lib/src/python_modules/messages.cpp

```diff
@@ -68,6 +70,22 @@ PYBIND11_MODULE(messages, m)
     // Allows python objects to keep DataTable objects alive
     py::class_<IDataTable, std::shared_ptr<IDataTable>>(m, "DataTable");
 
+    // EdgeConnectors for derived classes of MultiMessage to MultiMessage
+    srf::node::EdgeConnector<std::shared_ptr<morpheus::MultiInferenceMessage>,
+                             std::shared_ptr<morpheus::MultiMessage>>::register_converter();
+
+    srf::node::EdgeConnector<std::shared_ptr<morpheus::MultiInferenceFILMessage>,
+                             std::shared_ptr<morpheus::MultiMessage>>::register_converter();
+
+    srf::node::EdgeConnector<std::shared_ptr<morpheus::MultiInferenceNLPMessage>,
+                             std::shared_ptr<morpheus::MultiMessage>>::register_converter();
+
+    srf::node::EdgeConnector<std::shared_ptr<morpheus::MultiResponseMessage>,
+                             std::shared_ptr<morpheus::MultiMessage>>::register_converter();
+
+    srf::node::EdgeConnector<std::shared_ptr<morpheus::MultiResponseProbsMessage>,
+                             std::shared_ptr<morpheus::MultiMessage>>::register_converter();
+
```

Comment:
This will be cumbersome, but we need to define all of the possible combinations here. So if we have `MultiInferenceNLPMessage` -> `MultiInferenceMessage` and `MultiInferenceMessage` -> `MultiMessage`, then we need to define:

- `MultiInferenceMessage` -> `MultiMessage`
- `MultiInferenceNLPMessage` -> `MultiInferenceMessage`
- `MultiInferenceNLPMessage` -> `MultiMessage`